src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,42 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter DATE_TIME_FORMATTER
+            = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    private static final int ONE_DAY = 1;
+    private static final int DATE_INDEX = 0;
+    private static final int NAME_INDEX = 1;
+    private static final int HOURS_INDEX = 2;
+    private static final int SALARY_INDEX = 3;
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder builder = new StringBuilder("Report for period");
+        builder.append(" ").append(dateFrom).append(" - ").append(dateTo);
+        LocalDate from = LocalDate.parse(dateFrom, DATE_TIME_FORMATTER);
+        LocalDate fromNew = from.minusDays(ONE_DAY);
+        LocalDate to = LocalDate.parse(dateTo, DATE_TIME_FORMATTER);
+        LocalDate toNew = to.plusDays(ONE_DAY);
```

Comment:
we don't need ONE_DAY constant and these variables (toNew fromNew), just use isEqual method of localDate together with isAfter and isBefore